backend/src/model/dto.rs

```diff
@@ -175,3 +180,73 @@ pub struct Page<T> {
     /// Number of pages in total.
     pub total_pages: i32,
 }
+
+#[allow(clippy::missing_docs_in_private_items)] // TODO: See #97.
```

Comment:
Please document the structs and their fields. #97 is already close, so it should be referenced anywhere anymore.